### DISCUSSION TITLE
Whitelist senders marked as spam by SimpleLogin

### DISCUSSION THREAD

**Discussion by sawft99** [simple-login/app#1282] [Feature Requests]
Unfortunately I receive emails from some places that do things out of line with their spf, dmarc, etc, and SL sends it to spam, deservedly so. However, since these emails are reoccurring and the sender has yet to fix their issues. I have to login to SL to manually download their messages every single time. It would be nice if I could whitelist a specific sender for an alias to be allowed through the SL spam filter. Otherwise it is this manual checking for every email.

**Comment by WhamboMPS** [comment]
This is a no-brainer idea: hopefully it will be implemented.
